### DISCUSSION TITLE
Chia can't synchronize

### DISCUSSION THREAD

**Discussion by hadesya** [Chia-Network/chia-blockchain#4306] [Q&A]
Chia can't synchronize

**Comment by lvcivs** [comment]
Your node may not have enough connections. Try the recommendations here: https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#why-does-my-node-have-no-connections-how-can-i-get-more-connections

**Comment by usao-az** [comment]
Ive got 52 connections showing under my connections tab, yet my full node constantly goes out of sync. In addition, since I upgraded to 1.1.5 my Plots and Farm tab just have the spinning icon, no data at all for over 6 hours. Not sure what to do either.
